source/views/stoprint/print-jobs.tsx

```diff
@@ -100,13 +94,13 @@ class PrintJobsView extends React.PureComponent<Props, State> {
 
 	keyExtractor = (item: PrintJob) => item.id.toString()
 
-	openSettings = () => this.props.navigation.navigate('SettingsView')
+	openSettings = () => navigation.navigate('SettingsView')
```

Comment:
Where does `navigation` come from?

Reply:
I have a bullet point in the PR description but I should have made it a comment in the code.

> open settings button is broken here but fixed in the follow-up PR